code/modules/food_and_drinks/machinery/processor.dm

```diff
@@ -72,41 +82,41 @@
 	default_unfasten_wrench(user, tool)
 	return TOOL_ACT_TOOLTYPE_SUCCESS
 
-/obj/machinery/processor/attackby(obj/item/O, mob/living/user, params)
+/obj/machinery/processor/attackby(obj/item/attacking_item, mob/living/user, params)
 	if(processing)
 		to_chat(user, span_warning("[src] is in the process of processing!"))
 		return TRUE
-	if(default_deconstruction_screwdriver(user, "processor", "processor1", O))
+	if(default_deconstruction_screwdriver(user, "processor", "processor1", attacking_item))
 		return
 
-	if(default_pry_open(O))
+	if(default_pry_open(attacking_item))
 		return
 
-	if(default_deconstruction_crowbar(O))
+	if(default_deconstruction_crowbar(attacking_item))
 		return
 
-	if(istype(O, /obj/item/storage/bag/tray))
-		var/obj/item/storage/T = O
+	if(istype(attacking_item, /obj/item/storage/bag/tray))
+		var/obj/item/storage/attacking_storage = attacking_item
 		var/loaded = 0
-		for(var/obj/S in T.contents)
-			if(!IS_EDIBLE(S))
+		for(var/obj/content_item in attacking_storage.contents)
+			if(!IS_EDIBLE(content_item))
 				continue
-			var/datum/food_processor_process/P = PROCESSOR_SELECT_RECIPE(S)
-			if(P)
-				if(T.atom_storage.attempt_remove(S, src))
-					LAZYADD(processor_contents, S)
+			var/datum/food_processor_process/recipe = PROCESSOR_SELECT_RECIPE(content_item)
+			if(recipe)
+				if(attacking_storage.atom_storage.attempt_remove(content_item, src))
+					LAZYADD(processor_contents, content_item)
 					loaded++
 
 		if(loaded)
 			to_chat(user, span_notice("You insert [loaded] items into [src]."))
 		return
 
-	var/datum/food_processor_process/P = PROCESSOR_SELECT_RECIPE(O)
-	if(P)
-		user.visible_message(span_notice("[user] put [O] into [src]."), \
-			span_notice("You put [O] into [src]."))
-		user.transferItemToLoc(O, src, TRUE)
-		LAZYADD(processor_contents, O)
+	var/datum/food_processor_process/recipe = PROCESSOR_SELECT_RECIPE(attacking_item)
+	if(recipe)
+		user.visible_message(span_notice("[user] put [attacking_item] into [src]."), \
+			span_notice("You put [attacking_item] into [src]."))
```

Comment:
```suggestion
		user.visible_message(
			span_notice("[user] put [attacking_item] into [src]."),
			span_notice("You put [attacking_item] into [src]."),
		)
```

New and hip way to multiline stuff like this, may as well do it while you're in the area and cleaning up things.